doc/data/messages/d/duplicate-bases/details.rst

```diff
@@ -0,0 +1,7 @@
+.. code:: shell-session
```

Comment:
I didn't know this raised a `TypeError`. Could you add this to the description of `duplicate-bases`  in its declaration in https://github.com/PyCQA/pylint/blob/main/pylint/checkers/classes/class_checker.py?

We can then remove this file.

Reply:
Done ✔️